app/src/matchmaking/match/match.hpp

```diff
@@ -161,7 +161,7 @@ class Match {
 
     inline static constexpr char INSUFFICIENT_MSG[]     = "Draw by insufficient material";
     inline static constexpr char REPETITION_MSG[]       = "Draw by 3-fold repetition";
-    inline static constexpr char ILLEGAL_MSG[]          = " made an illegal move";
+    inline static constexpr char ILLEGAL_MSG[]          = " makes an illegal move";
```

Comment:
changed to be consistent with cutechess https://github.com/cutechess/cutechess/blob/1071d84cf272bd7deca0964336bf02e367e2b22b/projects/lib/src/board/result.cpp#L121